modules/sql/src/main/resources/db/migration/V063__Wavelength.sql

```diff
@@ -0,0 +1,47 @@
+--
+-- Adds comments to various tables, columns, types.
+--
+
+-- Switch GOMS to nm, with precision to distinguish pm 
+-- 10,3 can hold [0, 2147483647] pm as nm
+
+ALTER TABLE smart_gmos_north
+  ALTER COLUMN min_wavelength SET DATA TYPE numeric(10,3),
+  ALTER COLUMN max_wavelength SET DATA TYPE numeric(10,3);
```

Comment:
It's kind of annoying to see, for example, 300 nm appear as `300.000` in the database but clipping it to anything less means an arbitrary `Wavelength` won't roundtrip.

Reply:
This seems correct. Keeping stuff stable as it round-trips is really important.